src/platforms/linux/linuxdependencies.cpp

```diff
@@ -19,74 +19,9 @@
 #include "logger.h"
 
 namespace {
-
 Logger logger("LinuxDependencies");
-
-void showAlert(const QString& message) {
-  logger.debug() << "Show alert:" << message;
-
-  QMessageBox alert;
-  alert.setText(message);
-  alert.exec();
-}
-
-#ifndef MZ_FLATPAK
-static bool checkDaemonVersion() {
-  logger.debug() << "Check Daemon Version";
```

Comment:
Why don't we need this anymore?

Reply:
This check did two things:
1. Verify that the daemon is running and error out early if it's missing, and
2. Check that a valid `PATH` environment variable has been set.

Checking for the existence of the daemon doesn't have good error handling about it (it just halts the process and presents a popup error message), and now that the daemon recovery protocol exists it probably is better handled by just letting the controller handle errors normally and retry as needed.

We don't use the `PATH` anymore in the client. I think this is a holdover from before we even had a daemon and we invoked a script (`wg-quick`) to bring up the wireguard interface.